alvr/session/src/settings.rs

```diff
@@ -627,6 +627,20 @@ pub enum PassthroughMode {
     ChromaKey(#[schema(flag = "real-time")] ChromaKeyConfig),
 }
 
+#[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq, Debug)]
+pub struct UpscalingConfig {
+    pub edge_direction: bool,
+    #[schema(gui(slider(min = 1.0, max = 16.0, step = 1.0)))]
+    pub edge_threshold: f32,
+    #[schema(gui(slider(min = 1.0, max = 2.0, step = 0.01)))]
+    pub edge_sharpness: f32,
+    #[schema(gui(slider(min = 1.0, max = 3.0, step = 0.01)))]
+    #[schema(strings(
+        help = "In-headset dimensional resolution multiplier applied on top of render resolution"
```

Comment:
It would be good to say that increasing this too much will cause performance issues